### ISSUE TITLE
Apendices estão com a numeração errada.

### ISSUE THREAD

**Issue by marcoaaguiar** [eohulse/ufsc-thesis-latex-template#4]
Os apêndices estão aparecendo como seções do último capítulo quando referenciadas e não apresentam numeração no sumário e nem na primeira página da seção.

**Comment by eohulse** [comment]
Sim, essa funcionalidade ainda não está totalmente funcional e automática. Se possível ela será incorporada numa próxima atualização.

Provisóriamente aconselho utilizar o seguinte workaround para nomear e referenciar a Apêndices/Anexos:
1) Nomeie os apendices e anexos da seguinte maneira:
\chapter{Appendix A --- "nome do apêndice"} \label{chap:appendix-a}
2) Para referenciar ao apêndice/anexo no texto:
\hyperref[chap:appendix-a]{Appendix A}
